src/views/Profile/AuthorsList/AuthorsTableBody.tsx

```diff
@@ -32,12 +50,27 @@ const AuthorsTableBody: React.FC<IAuthorsTableBodyProps> = ({ authors }) => {
             {fullName}
           </Typography>
         </TableCell>
-        <TableCell>{region}</TableCell>
-        <TableCell>City</TableCell>
-        <TableCell>{dateOfCreation}</TableCell>
+        <TableCell>{region?.name}</TableCell>
```

Comment:
temporary solution. I added as an optional because as i understood for some of the test authors, fields 'region' and 'main Institution' equal to null. But while creating an author using basic logic such fields are required and they will always have valid values